src/picongpu/include/simulation_defines/param/gasConfig.param

```diff
@@ -249,7 +249,7 @@ struct FreeFormulaFunctor
      *
      * @return float_X density [normalized to 1.0]
      */
-    HDINLINE float_X operator()(const floatD_64& position_SI, const floatD_64& cellSize_SI)
+    HDINLINE float_X operator()(const floatD_64& position_SI, const float3_64& cellSize_SI)
```

Comment:
just a note: `cellSize_SI` is a debug parameter and now in different dimensionality so that even `.shrink<simDim>()` would be needed for usage, so we might remove or refactor that in the future since I don't like confusing the user with it.